src/components/section/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './style.css';
+
+export default class Section extends React.Component {
```

Comment:
I think all of our components could be made stateless except for the nav. I shall add a task to do this.